fern/docs/pages/cli/fern-generate.mdx

```diff
@@ -44,6 +44,17 @@ $ fern generate --version
 2.1.1
 ```
 
+### Intermediate Representation(IR) Version
+
+**Type**: `string`
+**Description:** Specify a version of Intermediate Representation(IR).
+
+**Example**
+
+```bash
+$ fern generate --irVersion v32
```

Comment:
@zdunecki looks great -- thoughts on calling this `--irVersionOverride` to signify that you are overriding typical CI behavior ? 

For example if you run the python generator with `--irVersion 33` even though it actually depends on 32, the CLI would now allow it